### DISCUSSION TITLE
Contribution to openAMP

### DISCUSSION THREAD

**Discussion by Raveem13** [OpenAMP/open-amp#479] [General]
Hi Team,
Is there any limitation or upcoming feature where I can contribute to the project?

Thanks.

**Comment by arnopo** [comment]
Hello @Raveem13 ,

Thanks for your proposal.

Please could you introduce yourself by giving an overview of your experiences with open-amp?

**Comment by Raveem13** [comment]
Hi @arnopo,
Thanks for the reply.
I'm a Master Student in Embedded systems. I worked on testing open-amp & building application using for different Xilinx family boards namely Zynq MP, Versal ACAP series.
Just wanted to know any room for the contribution to the project.

**Comment by arnopo** [comment]
Hi @Raveem13 
Sorry for the late answer.
I think @wmamills could be interested with some help on CI. Waiting that, you can have a look on the issue backlog such as 
https://github.com/OpenAMP/open-amp/issues/342, https://github.com/OpenAMP/open-amp/issues/343, https://github.com/OpenAMP/open-amp/issues/349. 
Except if @tammyleino  want to fix them by herself

**Comment by tammyleino** [comment]
@arnopo #342 looks like it needs more discussion, but @Raveem13 is welcome to work on the other two, or even #342 if he wants.
